tests/unittests/OperatorTest.cpp

```diff
@@ -3487,6 +3487,68 @@ TEST_P(InterpAndCPU, insertTensorTest) {
   }
 }
 
+/// Test RowwiseQuantizedFullyConnected Node.
+TEST_P(InterpOnly, rowwiseQuantizedFCTest) {
+  // In this test we subtract the outputs of a row-wise quantized FC and a
+  // floating-point FC and ensure that the error is below some low value.
+  auto *input =
+      mod_.createPlaceholder(ElemKind::FloatTy, {2, 100}, "in", false);
+  auto *fc = F_->createFullyConnected(ctx_, "FC", input, 5);
+
+  auto *weights = llvm::cast<Placeholder>(fc->getWeights());
+  auto *bias = llvm::cast<Placeholder>(fc->getBias());
+
+  ctx_.allocate(input)->getHandle().randomize(-1.0, 1.0, mod_.getPRNG());
+  ctx_.get(bias)->getHandle().randomize(0, 0.1, mod_.getPRNG());
```

Comment:
Same question :) isn't 0.1 too low comparing to other tensors? If you remove `sum += B` from the implementation, does the test fail?

Reply:
I remember the accuracy drop should be less then 2%... @rdzhabarov Please correct me if I am wrong. For the bias 0.1, if "sum+=b" is removed, the test failed -- actual: 4 vs 3. Actually, I tried to increase it to 1 and 2, but the result was pretty bad -- over 50% .  I also tried 0.01, 0.001, and the result is almost as good as "0.1".   From the experiment, I can say the the result is affected by all the params, but I don't know how :P

Reply:
So if bias is generated in range [0, 2), then we have a lot of differences between normal FC and RWQFC? Isn't this suspicious? One way to verify that our actual implementation is correct is to replicate this in Caffe2. Do we also see big divergence between FC and RWQFC in Caffe2 when bias is bigger?

Reply:
"So if bias is generated in range [0, 2), then we have a lot of differences between normal FC and RWQFC?"
--- Yes. For regular quantized FC, we has the similar bad result. So I think this is something related to params.

Reply:
> I remember the accuracy drop should be less then 2%... @rdzhabarov Please correct me if I am wrong

There are many factors which impact the %, like whether scale and offset are selected correctly to represent the data (we do not profile here, no fancy {s,o} selection based on distribution), type of the operation we validate. We should make test to pass, but be conservative with the threshold. No hard rule for 2%. I'd look at the data and choose what makes sense (if the value is too high then see if there are any bugs).